cmd/nerdctl/start.go

```diff
@@ -155,6 +165,11 @@ func startContainer(ctx context.Context, container containerd.Container, flagA b
 	if !flagA {
 		return nil
 	}
+	if flagA && flagT {
+		if err := tasks.HandleConsoleResize(ctx, task, con); err != nil {
```

Comment:
Why new `if flagA && flagT`, you already have the same condition [above](https://github.com/containerd/nerdctl/pull/1775/files#diff-0cc8445057142657a3937f86dc488b5283257fa971cbd4ac7546affa47fa1d53R125)

Reply:
My bad we need to catch a `task` before